### DISCUSSION TITLE
Bluetooth keyboard sleep

### DISCUSSION THREAD

**Discussion by Oogiee** [darthcloud/BlueRetro#949] [Q&A]
Have GC Bluetooth single port dongle and a Logitech K380s keyboard connected that I believe is just going into sleep mode every 25 seconds even if I'm actively typing. (using with phantasy star online) Is there a solution to this issue or do I need to find another keyboard to pair.

**Comment by darthcloud** [comment]
I'm using the 8bitdo keyboard these day without issue.

I'm sure your issue could be fix but I would need to have that KB myself.
